app/controllers/movies_controller.rb

```diff
@@ -15,4 +16,9 @@ def index
   end
 
   def discover; end
+
+  def show
```

Comment:
Nice skinny controller method here. I really like how you moved the functionality into the MovieData and MovieResult data.